security-release-process-documentation/security-release-process.md

```diff
@@ -258,3 +297,346 @@ Yes.
 CrashOverride will vouch for Acidburn joining the list on behalf of the "Seven"
 distribution.
 ```
+
+## Security Bug Bar
+
+The content presented below outline basic criteria when considering the effect
+of bugs disclosed to the Product Security Team and their severity. These apply
+mostly to items in "core" but could be abstractly applied to those in other
+repos as well.
+
+### Server
+
+The server bar is usually not appropriate when user interaction is part of 
+the exploitation process. If a "Critical" vulnerability exists only on server 
+components, and is exploited in a way that requires user interaction and results 
+in the compromise of the server, the severity may be reduced from "Critical" to 
+"Important" in accordance with the data definition of extensive user 
+interaction presented at the start of the client severity pivot.
+
+#### Critical
+
+Network worms or _unavoidable_ cases where the server is "owned."
+
+**Elevation of privilege** 
+
+The ability to either execute arbitrary code or obtain more privilege than 
+authorized. 
+
+- Remote Anonymous User
+    - Unauthorized access. Examples:
```

Comment:
Worth calling out Read or Write access to k8s API objects?

Reply:
will do!